### ISSUE TITLE
unknown error while connecting to github

### ISSUE THREAD

**Issue by B-a-a-S** [atom/github#1155]
### Prerequisites

* Have you tried to reproduce the problem in [Safe Mode][safe-mode]?
* Have you followed all applicable steps in the [debugging guide][debugging-guide]?
* Have you checked the FAQs on the [message board][message-board] for common solutions?
* Have you checked that your issue [isn't already filed][already-filed]?

[safe-mode]: http://flight-manual.atom.io/hacking-atom/sections/debugging/#using-safe-mode
[debugging-guide]: http://flight-manual.atom.io/hacking-atom/sections/debugging/
[message-board]: https://discuss.atom.io/c/faq
[already-filed]: https://github.com/atom/github/issues

### Description

[Description of the issue]

### Steps to Reproduce

1. [First Step]
2. [Second Step]
3. [and so on...]

**Expected behavior:**

[What did you expect to happen?]

**Actual behavior:**

[What actually happened instead?]

**Reproduces how often:**

[What percentage of the time does this happen?]

### Versions

You can get this information from copy and pasting the output of `atom --version` and `apm --version` from the command line. Also, please include the OS and what version of the OS you're running.

### Additional Information

Please include any additional information, configuration or data that might be necessary to reproduce the issue. Screenshots, if appropriate, are helpful as well.

If your issue involves a repository, and that repository is public, please include a link to it.

**Comment by rsese** [comment]
Thanks for the report - is the repository you're working on a private organization repository?  If so, you may be running into 3rd party OAuth App restrictions /cc https://github.com/atom/github/issues/1025#issuecomment-323140307.

**Comment by B-a-a-S** [comment]
Hi, 
It is my personal repository (not organization)

**Comment by rsese** [comment]
Is the repository private or public?  If it's a private repository, can you confirm if you have the same problem with a public repository?

**Comment by B-a-a-S** [comment]
Yes, I tried with both private and public repos

**Comment by rsese** [comment]
Hmmm, can you share what operating system you're on?  Also, when working with one of your public repositories, can you open dev tools and share what you see in the Console tab after entering your token?

**Comment by bigdgentry** [comment]
I have the same issue.  Now I cannot back out of the error on the GitHub tab.  I am on a Mac OS X Sierra

**Comment by rsese** [comment]
@bigdgentry are you working on an private organization repository?

**Comment by namlet** [comment]
I am also having this issue. Private repo.

**Comment by rsese** [comment]
@namlet is the repository owned by an organization?

**Comment by namlet** [comment]
@rsese Yes, and I read the thread above and finally got it working. Not very intuitive. Originally, when I authorized the app, there was a list of orgs and who had access. The only one that I was able to auth, was my account, and it looked different so I said OK. But when I had to go back to figure out why it wasn't working, I noticed that the box next to the org (WITH a green checkmark mind you) said "Grant". Not realizing this was a button or a step that needed to be accomplished, I had missed it during the first pass.

**Comment by rsese** [comment]
Great glad you have things sorted out @namlet - on the Atom side, there's an issue open about improving the "Unknown error occurred" error message https://github.com/atom/github/issues/922 if you want to subscribe.

**Comment by d3bugged** [comment]
me on parrot having the same issue. i have oauth permissions
